rslp/lfmc/README.md

```diff
@@ -1,51 +1,96 @@
 # Live Fuel Moisture Content (LFMC) Estimation
 
-The original LFMC work is based on Galileo model (more details can be found in Patrick J's [paper](https://www.overleaf.com/project/6819d1e76eb0ba2dde451e04), TODO: update link once it's opne-sourced) and the data preparation part is relying on Google Earth Engine. Here, we switch to rslearn for data preparation (right now, only the following modalities are supported: Sentinel2, Sentinel1, SRTM, location, we will need to add ERA5 and mayb TerraClimate in the future). The alation study of the paper (Table 9) indicates that adding ERA5 and TerraClimate help reduce the error in LFMC estimation.
+The original LFMC work is based on the Galileo model (more details can be found in Patrick J's [paper](https://arxiv.org/abs/2506.20132)), with data preparation relying on Google Earth Engine. Here, we switch to rslearn for data preparation, with two options for the model, SatlasPretrain and OlmoEarth (the latter is not open source yet).
 
-- LFMC ground-truth data: `/weka/dfive-default/yawenz/datasets/LFMC/lfmc-labels.csv`
-
-This file is the processed LFMC ground-truth data, which includes metadata like `latitude`, `longitude`, `sampling_date`, `site_name`, `lfmc_value`, `state_region`, `country`, `landcover` and `elevation`. We further cut off the LFMC value by 302 which is the 99.9% value.
+Right now, only the following modalities are supported: Sentinel2, Sentinel1, SRTM, and location. We will need to add ERA5 and maybe TerraClimate in the future -- the ablation study of the paper (Table 9) indicates that adding ERA5 and TerraClimate helps reduce the error in LFMC estimation.
 
 ### 1. Prepare Labels
 
-Run the following command to prepare labels:
+Steps 1-3 cover creating an rslearn dataset with the LFMC data. Alternatively you can
+download and extract the dataset from the tar URL below and skip to step 4. This
+dataset includes landcover and elevation tags on the windows that were obtained
+separately, to support breakdown of results by those categories.
+
+- https://storage.googleapis.com/ai2-rslearn-projects-data/datasets/lfmc/20250626.tar
+
+Run the following command to prepare labels (note that this script requires pandas to
+be additionally installed):
 ```
 python -m rslp.lfmc.prepare_labels --csv_path /tmp/lfmc-labels.csv
 ```
 
+This file is the processed LFMC ground-truth data, which includes metadata like `latitude`, `longitude`, `sampling_date`, `site_name`, `lfmc_value`, `state_region`, and `country`. We further cut off the LFMC value by 302 which is the 99.9% value.
+
 ### 2. Create Windows
 
-Run the following commands to create windows:
+Run the following commands to create windows (note that this script requires pandas to
+be additionally installed):
 ```
-python -m rslp.lfmc.create_windows --csv_path /tmp/lfmc-labels.csv --ds_path /weka/dfive-default/rslearn-eai/datasets/lfmc/20250626 --window_size 32
+export DATASET_PATH=/path/to/dataset
+export DATASET_GROUP=global_lfmc
```

Comment:
I know you copied this over, but let's use globe_lfmc for consistency with the original dataset name.